judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueriesWithConstantParameters.java

```diff
@@ -0,0 +1,388 @@
+     *    * s12.ffTimestamp!isUndefined()
+     *    * s12.ffLong!isDefined()      and s12.fLong == 9999999999
+     *    * s12.ffString!isUndefined()
+     *    * s12.ffDecimal!isUndefined()
+     *    * s12.ffEnum!isDefined()      and s12.fEnum == MyEnum#A02
+     *
+     *  // for fTimestamp and fString
+     *  . Create and save a MyEntity instance with the following field values. (*e4*)
+     *    * ffBool = true
+     *    * ffDate = `2023-01-01`
+     *    * ffTime = `12:00:13`
+     *    * ffTimestamp = `2020-01-01T14:11:12+01:00`
+     *    * ffLong = 1234567890
+     *    * ffString = "AAA"
+     *    * ffDecimal = 2.7109
+     *    * ffEnum = MyEnum#A01
+     *
+     *  // for fDecimal
+     *  . Create and save a MyEntity instance with the following field values. (*e5*)
+     *    * ffBool = false
+     *    * ffDate = `2023-01-05`
+     *    * ffTime = `12:00:13`
+     *    * ffTimestamp = `2020-01-01T01:11:13-12:00`
+     *    * ffLong = -1234567890
+     *    * ffString = "cbaaa"
+     *    * ffDecimal = 13.0001
+     *    * ffEnum = MyEnum#A03
+     *
+     *  . Create and save a Snapshot1 instance without any field values. (*s13*)
+     *
+     *  // Check the fTimestamp, fString and fDecimal
+     *  . Check the value of the fields of the created *s13* instance. All of the following boolean expressions must be true.
+     *    * s13.ffBool!isDefined()      and s13.fBool == true
+     *    * s13.ffDate!isDefined()
+     *    * s13.ffTime!isDefined()
+     *    * s13.ffTimestamp!isDefined() and s13.fTimstamp == `2020-01-01T14:11:12+01:00`
+     *    * s13.ffLong!isDefined()      and s13.fLong == 9999999999
+     *    * s13.ffString!isDefined()    and s13.fString == "AAA"
+     *    * s13.ffDecimal!isDefined()   and s13.fDecimal == 13.0001
+     *    * s13.ffEnum!isDefined()      and s13.fEnum == MyEnum#A02
+     *
+     *  . Retrieve the s21 Snapshot2 instance.
+     *
+     *  . All of the following boolean expressions must be true.
+     *    * s21.ffBool!isDefined()      and s21.fBool == true
+     *    * s21.ffDate!isDefined()
+     *    * s21.ffTime!isDefined()
+     *    * s21.ffTimestamp!isDefined() and s21.fTimstamp == `2020-01-01T14:11:12+01:00`
+     *    * s21.ffLong!isDefined()      and s21.fLong == 9999999999
+     *    * s21.ffString!isDefined()    and s21.fString == "AAA"
+     *    * s21.ffDecimal!isDefined()   and s21.fDecimal == 13.0001
+     *    * s21.ffEnum!isDefined()      and s21.fEnum == MyEnum#A02
+     *    * s21.entities!contains(e1) == false
+     *    * s21.entities!contains(e2) == false
+     *    * s21.entities!contains(e3) == true
+     *    * s21.entities!contains(e4) == false
+     *    * s21.entities!contains(e5) == true
+     *
+     *  . Retrieve the s12 entity from the database again.
+     *
+     *  . Check the value of the fields of the retrieved *s12* instance. All of the following boolean expressions must be true.
+     *    * s13.created!isDefined()
```

Comment:
I think it is a specification mistake. We retrive s12 entity from the database, and after that it would be logical to check the s12 entity values. This sentence confirms it: `Check the value of the fields of the retrieved *s12* instance. All of the following boolean expressions must be true.` . s12 entity values are already checked.